library/a-star/src/main/java/astar/AStar.java

```diff
@@ -121,15 +126,15 @@ public static Spot algo(Spot[][] grid, int x1, int y1, int x2, int y2) {
     }
 
     //manhattan distance formula
```

Comment:
euclidean now.

Reply:
i still keep manhattan, because i wish to run different performance tests.
I also want to look into those two fomulas:
 - g'(n) = 1 + alpha * (g(n) - 1)
 - h(n) = h'(n, w1) + distance(w1, w2) + h'(w2, goal)

as per https://theory.stanford.edu/~amitp/GameProgramming/Heuristics.html

Reply:
sounds good just saw that the current method being ran was euclidean